src/System.Data.SqlClient/src/System.Data.SqlClient.notsupported.cs

```diff
@@ -0,0 +1,894 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable CA2215,CS0114
```

Comment:
Should these have the comment indicating what they're suppressing?

Reply:
Those warnings already existed before in the generated code. Unsure if we want to document them.